app/views/layouts/_google_analytics_script.haml

```diff
@@ -1,73 +1,31 @@
 - if (APP_CONFIG.use_google_analytics)
-  <script type="text/javascript">
```

Comment:
Probably not a biggie, but removing this and using :javascript (instead of :plain) creates several script tags to the page <head>. While debugging this might look a bit weird.
":javascript Surrounds the filtered text with <script>[...]"